src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/ActionType.kt

```diff
@@ -49,3 +49,5 @@ data class ActionType<out T>(
 @JvmField val VIEW_BOARD = ActionType("View Board") { IssuesOnBoard(it) }
 @JvmField val LOG_IN = ActionType("Log In") { Unit }
 @JvmField val SET_UP = ActionType("Set Up") { Unit }
+@JvmField val VIEW_FOCUSED_COMMENT = ActionType("View Focused Comment") {}
```

Comment:
Nit: If we are changing the convention to not add `Unit` explicitly then we should remove it from other declarations.

Reply:
I'll stick to the existing convention.